front_end/css/styles.css

```diff
@@ -269,7 +269,7 @@ td > button {
 }
 
 .e-title-container {
-    width: 68%;
+    width: 63%;
```

Comment:
fixes a bug where the 'next exercise' button jumps from the right side of the screen to the left when the 'previous exercise' button appears and it gets edged out of room